src/application/registration/Routing.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  TransactionRouting,
+  TransactionsRouting,
+} from "../../domain/entities/TransactionRouting";
+import {
+  BASE_URL,
+  START_URL,
+  SUBMISSION_ID,
+  TRANSACTION_ID,
+} from "../../config/constants";
+import TransactionRegistrationType from "./TransactionRegistrationType";
+
+export const NAME_TEMPLATE = TransactionRegistrationType.NAME.toLowerCase();
```

Comment:
I did this to match the template name with the transaction type
But this can be done differently

The only trick is that when a get request is made, the controller extracts the last part of the url and uses it to find out the TransactionType and retrieve the correct routing.

Reply:
I changed the type to lowercase to avoid having to use the toLowercase and toUppercase method